public/src/main/java/com/revenuecat/purchases/models/PurchaseOption.kt

```diff
@@ -6,6 +6,13 @@ import android.os.Parcelable
  * A purchase-able entity for a subscription product.
  */
 interface PurchaseOption : Parcelable {
+    /**
+     * If this PurchaseOption represents a Google subscription base plan, this will be the base plan ID.
+     * If it represents an offer, it will be the offer ID.
+     * Not applicable for Amazon subscriptions.
+     */
+    val id: String
```

Comment:
Should we make it optional for Amazon or an OTP? Again, not sure how the API will look like in these cases, so it might not be needed...

Reply:
Let's leave it like this since we haven't determined how Amazon and OTP look like